crates/iota-names/src/config.rs

```diff
@@ -56,6 +56,17 @@ impl IotaNamesConfig {
         }
     }
 
+    pub fn from_env() -> anyhow::Result<Self> {
+        Ok(Self::new(
+            std::env::var("IOTA_NAMES_PACKAGE_ADDRESS")?.parse()?,
+            std::env::var("IOTA_NAMES_OBJECT_ID")?.parse()?,
+            std::env::var("IOTA_NAMES_PAYMENT_PACKAGE_ADDRESS")?.parse()?,
+            std::env::var("IOTA_NAMES_REGISTRY_ID")?.parse()?,
+            std::env::var("IOTA_NAMES_REVERSE_REGISTRY_ID")?.parse()?,
+            std::env::var("IOTA_NAMES_SUBDOMAIN_PROXY_PACKAGE_ID")?.parse()?,
```

Comment:
Actually this isn't needed anymore, but needs rebasing

Reply:
fixed